### PR TITLE
user/decibels: new package

### PR DESCRIPTION
**Pull request by tulilirockz**
Depends on TSC, so packaged typescript with it

### CODE REVIEW

File: user/decibels/template.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+]
+makedepends = ["gjs-devel"]
+depends = ["gjs", "libadwaita"]
+pkgdesc = "GNOME music player"
+maintainer = "tulilirockz <tulilirockz@outlook.com>"
+license = "GPL-3.0-or-later"
+url = "https://gitlab.gnome.org/GNOME/Incubator/decibels"
+source = [
+    f"{url}/-/archive/{pkgver}/decibels-{pkgver}.tar.gz",
+    "https://gitlab.gnome.org/BrainBlasted/gi-typescript-definitions/-/archive/dbbaa0527556cd3ce5434c4a5072cd99348eff7a/gi-typescript-definitions-dbbaa0527556cd3ce5434c4a5072cd99348eff7a.tar.gz",
```
Comment (nekopsykose) [inline user/decibels/template.py:22]:
```suggestion
    "https://gitlab.gnome.org/BrainBlasted/gi-typescript-definitions/-/archive/dbbaa0527556cd3ce5434c4a5072cd99348eff7a.tar.gz",
```

File: user/decibels/template.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "desktop-file-utils",
+    "gettext",
+    "libxml2-progs",
+    "meson",
+    "pkgconf",
+    "typescript",
+]
+makedepends = ["gjs-devel"]
+depends = ["gjs", "libadwaita"]
+pkgdesc = "GNOME music player"
```
Comment (nekopsykose) [inline user/decibels/template.py:16]:
```suggestion
pkgdesc = "GNOME audio player"
```

File: user/decibels/template.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "blueprint-compiler",
+    "desktop-file-utils",
+    "gettext",
+    "libxml2-progs",
+    "meson",
+    "pkgconf",
+    "typescript",
+]
+makedepends = ["gjs-devel"]
+depends = ["gjs", "libadwaita"]
```
Comment (nekopsykose) [inline user/decibels/template.py:15]: also gst-plugins-good i think (not sure which but something gst)

Comment (tulilirockz) [reply user/decibels/template.py]: from the GI definitions it pulls, apparently it only depends on gst-plugins-base

**Comment by tulilirockz** [comment]
Accidentaly deleted the remote branch when cleaning up my stuff

**Pull request merged**